assets/frag.glsl

```diff
@@ -205,8 +220,9 @@ vec4 ray_march(inout vec4 p, vec4 ray, float sharpness) {
 	float td = 0.0;
 	float min_d = 1.0;
 	for (; s < MAX_MARCHES; s += 1.0) {
-		if (d < MIN_DIST) {
-			s += d / MIN_DIST;
+		//if the distance from the surface is less than the distance per pixel we stop
+		if (abs(d) < FOVperPixel*td) {
```

Comment:
When zooming in, it was actually even slower for me than before!  Adding a minimum distance like this fixes the problem:
```
float min_dist = min(FOVperPixel*td, MIN_DIST);
if (abs(d) < min_dist ) {
    s += 0.14*d / min_dist ;
```

Reply:
OK, I added MIN_DIST. Though, technically, it was correct, if you assume the fractal had infinite details that is =)

Reply:
Sorry, this was a typo on my part.  Was supposed to be max() not min()

Reply:
It's strange that I didn't notice, I actually wrote it correctly like max() in my local version of the shader.

Reply:
Yeah, you also need to add min_dist in the scene function, without it you get the background color all over the place
![image_2019-02-06_13-54-18](https://user-images.githubusercontent.com/47035925/52339878-fe8fe100-2a16-11e9-90c4-ef2355bb94f2.png)